packages/components/breadcrumb/_breadcrumb.scss

```diff
@@ -16,10 +16,8 @@
 
 .nhsuk-breadcrumb {
   @include print-hide();
-
-  background-color: $color_nhsuk-white;
-  padding-bottom: 12px; /* [1] */
```

Comment:
Hi @pflynny  - could you look at the [1] and [2] comments at the top of this file and update things so things point to the right place? Thanks.